### PR TITLE
Feature/us2/side nav bar

### PR DESCRIPTION
**Pull request by dimitriosxmi**
Please take a moment to review this ❤️ 
```diff
- Delete public/next.svg
- Delete public/thirteen.svg
- Delete public/vercel.svg
+ Add components\svgs\BurgerIcon.svg
+ Add components\svgs\XIcon.svg

+ Add components\svgs\index.js (Exports the added svg's
+ Add components/SideBar/index.js
+ Add components/SideBarButton/index.js (to open/ close the sidebar)
! Update pages/index.js (refactored, applied new components, and added clickHandler)
```

[US2](https://github.com/dimitriosxmi/ArtistsReferenceOrganizer/issues/2)
[Deployment](https://artists-reference-organizer-ni6bew0np-dimitriosxmi.vercel.app/)

### CODE REVIEW

File: components/SideBar/index.js
Comment (nastiche) [inline components/SideBar/index.js:1]: Please Use Link component for links, buttons have different purpose (I had the same thing yesterday :-)

File: pages/index.js
Comment (nastiche) [inline pages/index.js:1]: Does it make sense to create Layout component and render it in the App.js already now? Or did you plan it for the next Sprint?

Comment (dimitriosxmi) [reply pages/index.js:1]: I haven't planned it at all 😄 I thought I wanna spare it unless the `index.js` page starts gets sortof overcrowded 😋

File: components/SideBar/index.js
Comment (dimitriosxmi) [reply components/SideBar/index.js:1]: Since I'm also doing more than just a navigation I'll try to adapt it as requested 😄

**Pull request merged**